src/components/JdsSearch/Search.stories.js

```diff
@@ -0,0 +1,64 @@
+import JdsSearch from './Search.vue'
+import { default as storybookMixin } from '../../utils/storybook'
+
+export default {
+  component: JdsSearch,
+  title: 'Components/Search',
+}
+
+const Template = (args, context) => {
+  return {
+    name: 'JdsSearchStories',
+    components: { JdsSearch },
+    props: Object.keys(context.argTypes),
+    mixins: [storybookMixin(args, context)],
+    template: `
+    <jds-search 
+      v-bind="$props"
+      v-on="events"
+    />
+    `,
+  }
+}
+
+const defaultArgs = {
+  icon: true,
+  button: true,
+  placeholder: 'My Placeholder'
+};
+
+export const Default = Template.bind({})
+Default.args = {
+  value: '',
```

Comment:
Hi @bangunbagustapa , I just noticed, it seems every story uses `value` property so I think it's better to include them inside `defaultArgs` like this:

```js
const defaultArgs = {
  icon: true,
  button: true,
  placeholder: 'My Placeholder',
  value: ''
};
```

so we can remove `value` props on every story because they already spread the `defaultArgs` object 😄

Reply:
Thanks, done [here](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/119/commits/9d82ca1026f0231fba35a0fad5b6bd70ee3f135b).